src/resonanceReconstruction/rmatrix/src/makeLegacyUnresolvedResonanceTable.hpp

```diff
@@ -3,13 +3,6 @@ makeLegacyUnresolvedResonanceTable(
     const ENDF::unresolved::EnergyDependent::JValue& endfParameters,
     const std::vector< double >& ) {
 
-  if ( endfParameters.INT() != 2 ) {
```

Comment:
Do the other values of `INT` change how things are processed? I saw this in the code and tried to figure out what I could change to help, but couldn't see what was unique to `INT=2`.

Reply:
I noted this in the pull request description. Resonance parameters are always interpolated using lin-lin interpolation in NJOY2016. The INT flag therefore only applies to how the resulting cross sections need to be interpolated. You should create a table with the unresolved energy points and the resulting cross sections, and linearise those using the INT flag.

Reply:
Oops, I skipped your comments. I just went right to the code.